src/org/usfirst/frc2175/command/group/RunIntakeInGroup.java

```diff
@@ -12,13 +13,20 @@
  */
 public class RunIntakeInGroup extends CommandGroup {
 
+    private double sideBeltSpeed;
+    private double mainBeltSpeed;
+
     public RunIntakeInGroup(DriverStation driverStation,
-            RobotSubsystems robotSubsystems) {
+            RobotSubsystems robotSubsystems, IntakeConfig intakeConfig) {
+        double sideBeltSpeed = intakeConfig.getSideBeltSpeedForward();
+        this.sideBeltSpeed = sideBeltSpeed;
+        double mainBeltSpeed = intakeConfig.getMainBeltSpeedForward();
+        this.mainBeltSpeed = mainBeltSpeed;
 
         addParallel(new RunDreamIntakeSideBeltAtSpeedCommand(driverStation,
-                robotSubsystems, 1));
+                robotSubsystems, sideBeltSpeed));
```

Comment:
Here, `sideBeltSpeed` refers to the the variable from line 21, not the one from line 16! You actually don't need to have lines 22 or 24. And you can delete the variables from lines 16 and 17.

You can also make these changes to RunIntakeOutGroup.